src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -207,6 +239,32 @@ public int getInt()
         return INTEGER_CONVERTER.convert(value == null ? defaultVal : value);
     }
 
+    /**
+     * Gets the value of a system property as a int.
+     * @return system property int value if it exists, overrideDefaultValue otherwise.
+     */
+    public int getInt(int overrideDefaultValue)
+    {
+        String value = System.getProperty(key);
+        if (value == null)
+            return overrideDefaultValue;
+
+        return INTEGER_CONVERTER.convert(value);
+    }
+
+    public void setInt(int value)
```

Comment:
Neutral opinion about this one, can have it, but also I can live without it :-)

Reply:
tests set it so use it to avoid 

```
System.setProperty(CassandraRelevantProperties.MY_PROP.getKey(), Integer.toString(42));
// vs
CassandraRelevantProperties.MY_PROP.setInt(42);
```